snforge_std/src/cheatcodes/tx_info.cairo

```diff
@@ -44,20 +44,18 @@ impl TxInfoMockImpl of TxInfoMockTrait {
     }
 }
 
-fn start_spoof(target: CheatTarget, tx_info_mock: TxInfoMock) {
-    let mut cheat_target_serialized: Array<felt252> = array![];
-    target.serialize(ref cheat_target_serialized);
-
-    let mut tx_info_serialized = array![];
-    tx_info_mock.serialize(ref tx_info_serialized);
-
-    let mut inputs: Array<felt252> = array![];
-    inputs.append_span(cheat_target_serialized.span());
-    inputs.append_span(tx_info_serialized.span());
-
+fn spoof(target: CheatTarget, tx_info_mock: TxInfoMock, span: CheatSpan) {
+    let mut inputs = array![];
+    target.serialize(ref inputs);
+    span.serialize(ref inputs);
```

Comment:
Missing taget + span validation

Reply:
[Add validation to spoof](https://github.com/foundry-rs/starknet-foundry/pull/1874/commits/6e6db5047be738fe979dd5fcca41a163ebf25339)